README.md

```diff
@@ -122,69 +88,76 @@ When you have completed this tutorial, you will understand how to:
+You need to complete the steps in this tutorial to deploy a CruchyDB Operator from Red Hat Marketplace on a OpenShift Cluster:
   - [Steps to Deploy CrunchyDB Operator](https://github.com/IBM/rhm-crunchydb-operator-install-steps)
   
-### Step 2: Perform CRUD Operations on CrunchyDB using python
-
-- Once we have the CrunchyDB UP and running, `user` and `database` created, we can now explore the CRUD operations on Crunchy PostgreSQL in a python runtime using Jupyter Notebook.
+### Step 2: Perform CRUD operations on CrunchyDB using Python
 
-- In Terminal run the following command to port forward `5432` port from the OpenShift cluster which we will be using in our Jupyter Notebook to establish a connection with the Crunchy PostgreSQL instance.
-```bash
-$ kubectl port-forward -n pgo svc/hippo 5432:5432 
-```
+Once you have the CrunchyDB Operator up and running and you've created a user and database, you can now explore the CRUD operations on Crunchy PostgreSQL in a Python runtime using a Jupyter Notebook.
 
-```
-Forwarding from 127.0.0.1:5432 -> 5432
-Forwarding from [::1]:5432 -> 5432
-```
+1. In your terminal, run the following command to port forward the `5432` port from the OpenShift cluster. This port is used in our Jupyter Notebook to establish a connection with the Crunchy PostgreSQL instance.
 
-- We will be working with Jupyter Notebook, you can use tools like [Anaconda](https://www.anaconda.com/products/individual) to open the Jupyter Notebook or [install Jupyter Notebook from python-pip](https://jupyter.org/install).
+    ```bash
+    $ kubectl port-forward -n pgo svc/hippo 5432:5432 
+    ```
 
-- Download and open the notebook [CrunchyDB CRUD Operations.ipynb](CrunchyDB%20CRUD%20Operations.ipynb) in your local machine.
+    ```
+    Forwarding from 127.0.0.1:5432 -> 5432
+    Forwarding from [::1]:5432 -> 5432
+    ```
 
-- Click on the **Cell** tab and click on **Run All** as shown.
+1. Download and open the Jupyter notebook [CrunchyDB CRUD Operations.ipynb](CrunchyDB%20CRUD%20Operations.ipynb) on your local machine.
 
-![](doc/source/images/run.png)
+1. Click on the **Cell** tab and select **Run All**.
 
-- You can now follow the notebook instructions for more details on what is happening in each cell.
+    ![](doc/source/images/run.png)
 
-- After we have execuited the notebook, we can verify the table in the pgAdmin 4 console. 
+    You can now follow the notebook instructions for more details on what is happening in each cell.
 
-- - To access pgAdmin 4, you can set up a `port-forward` to the Service, which follows the pattern `<clusterName>-pgadmin`, to port `5050`:
-```bash
-$ kubectl port-forward -n pgo svc/hippo-pgadmin 5050:5050 
-```
+1. After you execute the Jupyter notebook, you can verify the table in the pgAdmin 4 console. To access pgAdmin 4, you can set up a `port-forward` to the service<!--EM: What service? And what is pgAdmin 4??-->, which follows the pattern `<clusterName>-pgadmin`, to port `5050`:
```

Comment:
Port forward to CrunchyDb service(Console). pgAdmin4 is an  application to manage Postgres crunchy databases.